tests/cupy_tests/cuda_tests/test_memory.py

```diff
@@ -140,14 +140,30 @@ def test_free_all_free(self):
 class TestMemoryPool(unittest.TestCase):
 
     def setUp(self):
-        self.pool = memory.SingleDeviceMemoryPool()
+        self.pool = memory.MemoryPool()
 
     def test_zero_size_alloc(self):
-        mem = self.pool.malloc(0).mem
-        self.assertIsInstance(mem, memory.Memory)
-        self.assertNotIsInstance(mem, memory.PooledMemory)
+        from cupy.cuda import Device
```

Comment:
Please import at the head of source.
And, don't import objects, only modules.

We use PEP8 and a part of OpenStack Style Guidelines related to general coding style as our basic style guidelines.
see http://docs.openstack.org/developer/hacking/#imports